app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -95,6 +95,7 @@ class EventsFragment : Fragment() {
         showNoInternetScreen(isNetworkConnected())
 
         rootView.retry.setOnClickListener {
+            eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
```

Comment:
check for network connect if present then call this

Reply:
Retry button is visible only when network connection is not there